source/common/grpc/status.cc

```diff
@@ -85,5 +88,40 @@ uint64_t Utility::grpcToHttpStatus(Status::GrpcStatus grpc_status) {
   }
 }
 
+absl::optional<Status::GrpcStatus> Utility::nameToGrpcStatus(const std::string& grpc_status_name) {
```

Comment:
Yeah I would definitely add tests that fail if the invariance is not held. I would look at the response flag mappings for inspiration

Reply:
Will do. Where should it be located? `//tests/common/grpc/status_test.cc`?

Reply:
Test added in cab43ed.